code/game/atoms_movable.dm

```diff
@@ -202,8 +202,12 @@
 	step(pulling, get_dir(pulling.loc, A))
```

Comment:
Step returns false if the mob was unable to move. It could be a lot cleaner to add the cooldown in here instead by returning the return value of step and then doing.
```
if (!Move_Pulled())
  changeNext_move(CLICK_CD_MELEE)
```